software/flink.yml

```
@@ -0,0 +1,13 @@
+name: "Flink"
+website_url: "https://gitlab.com/rtraceio/web/flink"
+source_code_url: "https://gitlab.com/rtraceio/web/flink"
+description: "Create QR Codes, embeddable link previews for your website and crawls/scrapes metadata."
+licenses:
+  - MIT
+platforms:
+  - C#
+  - Docker
+  - K8S
```

```suggestion
platforms:
  - Docker
```

Can't find the docs for these deployment methods.

---

Hey @nodiscc - thanks for the suggestion.

I have a question though. `C#` is a programming language, not a "Platform". You can not deploy something to `C#`. So if `platforms` is meant to be only a deployment target, then `C#` (and any other programming language). If `platforms` is meant more like a "Tech Stack", then I'd love to keep `C#`, because `Flink` is written with C#.

Also when it comes to Kubernetes (K8s) - I have made it more visual that Flink is meant to be hosted on K8s too.
See: https://gitlab.com/rtraceio/web/flink/-/blob/main/docs/install/install.md?ref_type=heads#kubernetes-deployment-with-sqlite

Thanks

---

> made it more visual that Flink is meant to be hosted on K8s

:+1: 

> then I'd love to keep C#, because Flink is written with C#.

>> Values for platform are the main server-side requirements for the software.